mock/cmd/cmg/pkg/parse/package.go

```diff
@@ -80,6 +82,14 @@ func (iv *interfaceVisitor) Visit(node ast.Node) ast.Visitor {
 		switch t := n.Type.(type) {
 		case *ast.InterfaceType:
 			iv.interfaces = append(iv.interfaces, newInterface(iv.p, iv.file, n, t))
+		default:
+			if n.Assign != token.NoPos {
+				underlying := iv.p.TypesInfo.Types[t].Type.Underlying()
+				switch u := underlying.(type) {
```

Comment:
Just curious: why not
```go
if u, ok := underlying.(*types.Interface); ok {
...
```

Reply:
I guess either is fine for this case; it's already inside of two type switches. 🤣

Reply:
1 line >> 2 lines :P

Reply:
Yep, cleaned up this inner one and the outer one.